### ISSUE TITLE
Tradier Integration

### ISSUE THREAD

**Issue by ja2cool** [ninjaspread/ns#5]
Does this still exist, and if so how to access it?

**Comment by ninjaspread** [comment]
Yes. If you would like to enable it, please send me an email to admin@ninjaspread.com
